lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx/Balance.hs

```diff
@@ -902,14 +904,16 @@ balanceTransactionWithSelectionStrategyAndNoZeroAdaAdjustment
         -> ExceptT (ErrBalanceTx era) m ()
     guardWalletUTxOConsistencyWith u' = do
         let W.UTxO u = toWalletUTxO (recentEra @era) $ fromCardanoApiUTxO u'
-        let conflicts = lefts $ flip map (Map.toList u) $ \(i, o) ->
+        let conflicts = lefts $ flip map (Map.toList u) $ \(i, out1) ->
```

Comment:
Changing `o` to `out` breaks the visual similarity with `i`. Is this really worth it?

Reply:
Here I was aiming for visual similarity between `out1` and `out2`:
```hs
                   Just out2 -> unless (out1 == out2) $ Left
                        ( toLedgerTxOut era out1
                        , toLedgerTxOut era out2
                        )
```

I can rename them to `o1` and `o2` if you prefer?

Reply:
I'd prefer sticking to `o`, `o'`, or possibly `o1` or `o2` if you want them to have the same length

Reply:
Although the added descriptiveness might be worth it as you have it 🤔

Reply:
Okay! Fixed in 6e3879e6acb184eaa40b20a057fbeae60266d760.